concordia/settings_prod.py

```diff
@@ -78,3 +78,5 @@
 REGISTRATION_SALT = "django_registration"  # doesn't need to be secret
 
 SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
+
+RATELIMIT_BLOCK = os.getenv("RATELIMIT_BLOCK", True)
```

Comment:
Doesn’t this need to evaluate that to a Boolean? Maybe just have something like `getenv(…) != “false”`?

Reply:
I was thinking we'd set it to 0 or 1 in the env vars

Reply:
Okay, but don’t we still need to convert it from a string so it won’t always be true?

Reply:
Like this? `   RATELIMIT_BLOCK = bool(strtobool(str(os.getenv("RATELIMIT_BLOCK"))))` or you have a different preferred method?